run.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+aws sso login --profile PowerUserAccess-043802220583-SSO
+
+sudo docker build -t terratrain:latest .
+docker run --rm -it -v ~/.aws:/terraTrain/.aws terratrain
```

Comment:
Quick run, first run you will do with the full run.sh and the second run and every each after the first one you will perform with commented docker build, you can comment it straight away and just run without it, it will be easier to mount volume and do sso login with this kind of sh file instead of manual typing.